docs/topics/con_crud-design-tradeoffs.adoc

```diff
@@ -9,5 +9,5 @@
 a| * Each runtime determines how  to implement the database interactions. One can use a low-level connectivity API such as JDBC, some other can use JPA, and yet another can access ORM APIs directly. Each runtime decides what would be the best way.
 
  * Each runtime determines how the schema is created.
-a| * The PostgreSQL database example provided with this mission is not backed up with persistent storage. Changes to the database are lost if you stop or redeploy the database pod. To use an external database with your mission's pod in order to preserve changes, see the link:https://docs.openshift.com/online/dev_guide/integrating_external_services.html[Integrating External Services chapter] of the OpenShift Documentation. It is also possible to set up persistent storage with database containers on OpenShift. (For more details about using persistent storage with OpenShift and containers, see the link:https://docs.openshift.com/online/architecture/additional_concepts/storage.html[Persistent Storage], link:https://docs.openshift.com/online/dev_guide/volumes.html[Managing Volumes] and link:https://docs.openshift.com/online/dev_guide/persistent_volumes.html[Persistent Volumes] chapters of the OpenShift Documentation).
+a| * The PostgreSQL database example provided with this example is not backed up with persistent storage. Changes to the database are lost if you stop or redeploy the database pod. To use an external database with your example application's pod in order to preserve changes, see the link:https://docs.openshift.com/online/dev_guide/integrating_external_services.html[Integrating External Services chapter] of the OpenShift Documentation. It is also possible to set up persistent storage with database containers on OpenShift. (For more details about using persistent storage with OpenShift and containers, see the link:https://docs.openshift.com/online/architecture/additional_concepts/storage.html[Persistent Storage], link:https://docs.openshift.com/online/dev_guide/volumes.html[Managing Volumes] and link:https://docs.openshift.com/online/dev_guide/persistent_volumes.html[Persistent Volumes] chapters of the OpenShift Documentation).
```

Comment:
The PostgreSQL database example provided with this example is not backed up--> The PostgreSQL database example provided with this example application is not backed up